tensordict/tensordict.py

```diff
@@ -1717,7 +1722,11 @@ def permute(
         )
 
     def __repr__(self) -> str:
+        if self._being_called:
```

Comment:
`_being_called` is used in repr only? Then why not `_being_repr` or smth like it?

Reply:
Oh, I've just used same var name as you proposed here https://github.com/pytorch-labs/tensordict/issues/106, but sure lets rename it properly.